config/project/entryTypes/orgPerson--dd80a6fe-320a-4fb6-889d-17125d275aac.yaml

```diff
@@ -108,11 +108,11 @@ fieldLayouts:
           -
             dateAdded: '2025-04-17T04:38:10+00:00'
             elementCondition: null
-            fieldUid: 11127416-2baf-40e9-bb28-9ea76040af2a # Person Roles
-            handle: roles
+            fieldUid: 11127416-2baf-40e9-bb28-9ea76040af2a # Org Person Roles
+            handle: personRoles
```

Comment:
Any reason why we’re going with `entry.personRoles` here instead of the simpler `entry.roles` field handle?